sympy/polys/tests/test_constructor.py

```diff
@@ -121,3 +121,18 @@ def test_precision():
     result = construct_domain([f2])
     y = result[1][0]
     assert y-1 > 1e-50
+
+def test_issue_11538():
+    dom = ZZ[E]
+    assert construct_domain(E)[0] == ZZ[E]
+    dom = ZZ[x,E]
+    assert construct_domain(x**2 + 2*x + E) == \
```

Comment:
This can be replaced with `assert construct_domain(x**2 + 2*x + E)[0] == ZZ[x, E]`. The `assert` statements below can be treated similarly. None of the `dom` assignments are needed and they can be omitted.